### PR TITLE
Fix report behavior bug

### PR DESCRIPTION
**Pull request by Moonlit-Sailor**
Fix report folder behavior bug and test "wordcount" report behavior.

### CODE REVIEW

File: bin/functions/load_config.py [hunk 1]
```diff
@@ -14,11 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import sys
-import glob
-import re
-import urllib
-import socket
+import sys, os, glob, re, urllib, socket
```
Comment (carsonwang) [inline bin/functions/load_config.py]: What is the style suggested by PEP8? @gczsjdy

File: bin/functions/load_config.py [hunk 2]
```diff
@@ -185,12 +181,17 @@ def override_conf_by_paching_conf():
...
         workload_config_file,
-        workload_name,
+        workload_folder,
         patching_config=""):
     abspath = os.path.abspath
     conf_root = abspath(conf_root)
     workload_config_file = abspath(workload_config_file)
 
+    # get current workload job name and store it in workload_job
+    (dir,workload_job) = os.path.split(workload_folder)
```
Comment (carsonwang) [inline bin/functions/load_config.py]: `workload_job` is acutally the framework name (spark, hadoop, prepare) right? Can we rename it to `framework_name` although `prepare` is not a framework.

Comment (Moonlit-Sailor) [inline bin/functions/load_config.py]: Yeah, workload_job is framework name such as spark, hadoop, prepare, etc.
Ok, I will rename it.

File: bin/functions/load_config.py [hunk 3]
```diff
@@ -736,10 +737,10 @@ def export_config(workload_name):
     if len(sys.argv) < 4:
         raise Exception(
             "Please supply <conf root path>, <workload root path>, <workload folder path> [<patch config lists, seperated by comma>")
-    conf_root, workload_config, workload_name = sys.argv[
+    conf_root, workload_configFile, workload_folder = sys.argv[
```
Comment (carsonwang) [inline bin/functions/load_config.py]: The original style seems to be strange here? @gczsjdy, can you check all similar occurrences later in another PR ?
PEP8 limits all lines to a maximum of 79 characters. May be we can make it to 100 characters as an exception, just like Spark's python code.

**Pull request merged**